features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,7 @@
+Given(/^I visit the page "([^"]*)"$/) do |page|
+  visit page
+end
```

Comment:
I think it does, or? At least the step makes the test code see the text "Protected area" in the next step.